inlineplz/main.py

```diff
@@ -112,6 +112,8 @@ def inline(args):
         )
         if my_interface.post_messages(messages, args.max_comments) and not args.zero_exit:
             return 1
+
+        my_interface.clear_outdated_messages()
```

Comment:
Yeah, I assumed if its possible that we are hitting a reason which we would exit non-zero, and posting messages had a problem for the current commit, then clearing the outdated messages might not be what we want to do.